requirements.txt

```diff
@@ -151,4 +154,3 @@ Werkzeug==0.16.0
 wrapt==1.11.1
 zipp==3.5.0
 pyacoustid==1.2.2
```

Comment:
I think the sentry imports are still required (Devin just added them) and it looks like missing sentry imports are failing the CI?